netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -226,7 +256,7 @@ public void initChannel(Channel ch) {
         ServerTransportListener transportListener;
         // This is to order callbacks on the listener, not to guard access to channel.
         synchronized (NettyServer.this) {
-          if (channel != null && !channel.isOpen()) {
+          if (terminated) {
             // Server already shutdown.
```

Comment:
terminated is slightly different from shutdown. `shutdown` means no more new stream is allowed but the existing streams can still be active until they finish. `terminated` is once `shutdown` _and_ all streams are closed.  Here the comment `// Server already shutdown` seems to have been not accurate, whereas the old comment [here L283](https://github.com/grpc/grpc-java/pull/7674/files#diff-4cf42ed83c97f1bb5a16f70fce7d0a6719bb57c68e31a6c27aa2336fc5da53aeL283) `// Already closed` seems to be correct.